docs/reference-guides/data/data-core-notices.md

```diff
@@ -85,6 +126,21 @@ _Returns_
 
 Returns an action object used in signalling that a notice is to be created.
 
+_Usage_
+
+```js
+const ExampleComponent = () => {
+	const { createNotice } = useDispatch( 'core/notices' );
+	return (
+		<Button
+			onClick={ () => createNotice( 'success', __( 'Notice message' ) ) }
+		>
+			{ __( 'Generate a success notice!' ) }
```

Comment:
It would be nice to include all the required imports in this example, so `useDispatch` and `__`. Also, the preferred way of calling `useDispatch` is by passing the imported `store` object as follows:

```js
import { store as noticesStore } from '@wordpress/notices';
// ...
const { createNotice } = useDispatch( noticesStore );
```

Reply:
Thanks! I love importing the store name suggestion. All examples are updated.